src/Client.php

```diff
@@ -102,62 +106,65 @@ function getAllProducts(){
      * @param int $id Id of the product to retrieve.
      * @return object|false The product, if found. False if there's no product with that id.
      */
-    function getProduct($id){
-        $return = $this->callApi("Product_GetById",array("ProductId"=>$id));
+    public function getProduct($id){
+        $return = $this->callApi('Product_GetById',array('ProductId' =>$id));
         if(isset($return->Id)){
-            $product = new Product($return);
-            return $product;
+            return new Product($return);
         }
         return false;
     }
 
     /**
      * Retrives a product
-     * 
-     * @param int $id Id of the product you want to retrieve.
-     * @return object of found Products.
+     *
+     * @param array $ids
+     * @return array|null of found Products.
      */
-    function getProductsByIds(array $ids) : ?array {
-        return $this->callApi("Product_GetByIds", ["ProductIds"=>implode(",", $ids)]);
+    public function getProductsByIds(array $ids) {
```

Comment:
This will break backwards compatibility. Either implement so it takes an array OR comma seperated string, or wait with this until I have proper versioning going.